text/CinnabarLabFossilRoom.asm

```diff
@@ -11,19 +11,19 @@ _Lab4Text_75dc6::
 	line "fossil for me?"
 	prompt
 
-_Lab4Text_75dcb::
+_CinnabarLabFossilRoomScientist1NoFossilsText::
 	text "No! Is too bad!"
 	done
 
-_Lab4Text_75dd0::
+_CinnabarLabFossilRoomScientist1GoForAWalkText::
 	text "I take a little"
 	line "time!"
 
 	para "You go for walk a"
 	line "little while!"
 	done
 
-_Lab4Text_75dd5::
+_CinnabarLabFossilRoomScientist1FossilIsBackToLifeText::
```

Comment:
There are some other unnamed text labels in this file which are referenced by engine/events/cinnabar_lab.asm. They should get meaningful names as well, and be moved into their own file (like pokede_xratings.asm) since they aren't directly referenced by CinnabarLabFossilRoom scripts.

Reply:
I don't think it's useful to move this text to a separate file.

It's already co-located with all the other text for this map.
And even though scripts/CinnabarLabFossilRoom.asm doesn't directly reference these texts, it does *indirectly* reference them. (The routines in engine/events/cinnabar_lab.asm are called directly by scripts/CinnabarLabFossilRoom.asm)
This is pretty common for a lot of the misc things in engine/events/.

Reply:
I see. Then we may want to merge pokedex_ratings.asm back into OaksLab.asm, if the consistent alternative is to split out even more event-specific text files.

Reply:
No, I think text/pokedex_ratings.asm is fine as-is, because it is also leveraged by engine/menus/oaks_pc.asm and engine/movie/hall_of_fame.asm -- not just scripts/OaksLab.asm.